plugins/jobs/server/models/job.py

```diff
@@ -52,19 +56,33 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
-    def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
+    def list(self, user, types=None, statuses=None,
```

Comment:
Originally, the user parameter can only be a user dict or `None` to filter by user or anonymous job. I added a separate `listAll()` method to list all jobs when I implement the admin jobs page. However, when I create the `jobsmeta`(Now called JobsTypesAndStatuses), I feel it too redundant to have a series of a separate list and listAll on the model, so I combined them and used string `'all'` to indicate the difference. I tried to make it more clear. I thought `'none'` is better than `None` semantically, especially when put aside with  `'all'`. 
So, maybe I keep the `"none"` and add back `None` and make them behave the same?